iothub/service/src/Registry/Models/Job.cs

```diff
@@ -0,0 +1,67 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Newtonsoft.Json;
+
+namespace Microsoft.Azure.Devices
+{
+    /// <summary>
+    /// Contains the properties set by the service import/export job.
+    /// </summary>
+    public abstract class Job
+    {
+        /// <summary>
+        /// Creates an instance of this class. Provided for unit testing purposes only.
+        /// </summary>
+        internal Job()
+        { }
+
+        /// <summary>
+        /// The unique Id of the job.
+        /// </summary>
+        /// <remarks>
+        /// This value is created by the service. If specified by the user, it will be ignored.
+        /// </remarks>
+        [JsonProperty(PropertyName = "jobId", NullValueHandling = NullValueHandling.Ignore)]
+        public string JobId { get; internal set; }
+
+        /// <summary>
+        /// When the job started running.
+        /// </summary>
+        /// <remarks>
+        /// This value is created by the service. If specified by the user, it will be ignored.
+        /// </remarks>
+        [JsonProperty(PropertyName = "startTimeUtc", NullValueHandling = NullValueHandling.Ignore)]
+        public DateTime? StartedOnUtc { get; internal set; }
```

Comment:
Can this be DateTimeOffset?

Reply:
I realize this is not part of your PR, so something for us to think about independently.